engine-rest/engine-rest/src/test/java/org/camunda/bpm/engine/rest/JobDefinitionRestServiceInteractionTest.java

```diff
@@ -1319,18 +1333,43 @@ public void testSetJobRetries() {
     .when()
       .put(JOB_DEFINITION_RETRIES_URL);
 
-    verify(mockManagementService).setJobRetriesByJobDefinitionId(MockProvider.EXAMPLE_JOB_DEFINITION_ID, MockProvider.EXAMPLE_JOB_RETRIES);
+    verify(mockManagementService, times(1)).setJobRetries(MockProvider.EXAMPLE_JOB_RETRIES);
+    verify(mockSetJobRetriesBuilder, times(1)).jobDefinitionId(MockProvider.EXAMPLE_JOB_DEFINITION_ID);
+    verify(mockSetJobRetriesBuilder, times(1)).dueDate(null);
+    verify(mockSetJobRetriesBuilder, times(1)).execute();
+    verifyNoMoreInteractions(mockSetJobRetriesBuilder);
   }
 
   @Test
-  public void testSetJobRetriesExceptionExpected() {
-    String expectedMessage = "expected exception message";
+  public void testSetJobRetriesWithDueDate() {
+    Map<String, Object> retriesVariableJson = new HashMap<>();
+    retriesVariableJson.put("retries", MockProvider.EXAMPLE_JOB_RETRIES);
+    Date newDueDate = new Date(0);
```

Comment:
Similar to what was [mentioned by Tassilo](https://github.com/camunda/camunda-bpm-platform/pull/3060#pullrequestreview-1285141451) in the Java API PR, let's try to avoid `new Date(0)`.